pages/common/cmatrix.md

```diff
@@ -3,18 +3,18 @@
 > Shows a scrolling Matrix like screen in the terminal.
 > More information: <https://github.com/abishekvashok/cmatrix>.
 
-- Enable asynchronous scrolling:
+- Enable [a]synchronous scrolling:
 
 `cmatrix -a`
 
-- Display red text:
+- Change the text [C]olor (green by default):
 
 `cmatrix -C {{red}}`
 
-- Enable rainbow mode:
+- Enable [r]ainbow mode:
 
 `cmatrix -r`
 
-- Set screen update delay to 2 centiseconds (20 milliseconds):
+- Use a 1 second screen [u]pdate delay:
```

Comment:
As a quick reference page, I don't think we need to make the unit that explicit, given that centiseconds isn't a so common unit. IMHO the 100 value in the example command itself makes the unit clear enough.

Reply:
Putting 5 words more in the description to make it more clear doesn't hurt. I know this is `tldr`, but we shouldn't obsess over boiling everything down to the shortest form possible, sacrificing some valuable information in the process.

> given that centiseconds isn't a so common unit

Then I'd feel even more inclined to explain it there, as it may confuse someone.